6/changelog.yaml

```diff
@@ -86,3 +86,9 @@ releases:
 
         `Porting Guide <https://docs.ansible.com/ansible/devel/porting_guides.html>`_'
     release_date: '2022-08-02'
+  6.3.0:
+    changes:
+      deprecated_features:
+      - The google.cloud collection is considered unmaintained and will be
+        removed from Ansible 8
```

Comment:
Hmm, I'm wondering whether we should expand this to explain that this is not a final decision, but just part of the process (https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection).

Reply:
@felixfontein I was pondering myself if we should do this. How about `will be removed from Ansible 8 if nobody starts maintaining it again until then`? This makes clear that it's not a final decision.

I'm not sure if we should mention the exact process here, though.

Reply:
@felixfontein What do you think about this / my last commit?

Reply:
Looks good! I've added a suggestion with a link to the process description.